ui/src/components/Canvas.tsx

```diff
@@ -200,15 +241,19 @@ function ResultBlock({ pod, id }) {
           {pod.result.html ? (
             <div dangerouslySetInnerHTML={{ __html: pod.result.html }}></div>
           ) : (
-            pod.result.text && (
-              <Box>
-                <Box sx={{ display: "flex" }} bgcolor="lightgray">
-                  Result: [{pod.result.count}]:
-                </Box>
-                <Box>
-                  <Box component="pre" whiteSpace="pre-wrap">
-                    {pod.result.text}
-                  </Box>
```

Comment:
We need to show pod.result.text. In general, when a code block executes in Jupyter, the value of the last expression should be printed. like this:

<img width="367" alt="Screenshot 2022-11-20 at 4 25 36 AM" src="https://user-images.githubusercontent.com/4576201/202901766-e7432652-533c-4ab1-b4d1-3e6e71495714.png">

Reply:
cc @forrestbao

Reply:
I was wondering on this too. At first I thought this was due to that CodePod does not return the evaluation of the last expression.